### PR TITLE
Content moderator changes

### PR DESCRIPTION
**Pull request by v-sodsou**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/cognitiveservices/data-plane/ContentModerator/readme.md [hunk 1]
```diff
@@ -51,18 +51,18 @@ python:
   license-header: MICROSOFT_MIT_NO_VERSION
   add-credentials: true
   payload-flattening-threshold: 2
-  namespace: azure.cognitiveservices.vision.contentmoderator
-  package-name: azure-cognitiveservices-vision-contentmoderator
+  namespace: azure.cognitiveservices.contentmoderator
```
Comment (lmazuel) [inline specification/cognitiveservices/data-plane/ContentModerator/readme.md]: As discussed in the email thread, advertising of Content Moderator is under Vision. We need a clear justification to do not follow the official branding for this.
Not too that this package is already plugged in the Python repo in the automatic update system:
https://github.com/Azure/azure-sdk-for-python/tree/master/azure-cognitiveservices-vision-contentmoderator

so changing this breaks the SDK repo. Even if finally we rename the package, this PR is not the good place and should be done in sync with me (Python SDK owner and published)

**Review by lmazuel — CHANGES_REQUESTED** [review]
This PR is not the right place for Python conf update, see comment.

**Review by alvadb — APPROVED** [review]

**Comment by v-sodsou** [comment]
@lmazuel I have reverted my changes in readme file. Could you please take a look.

Thanks
Sonali Dsouza

**Review by lmazuel — APPROVED** [review]
LGTM

**Review by jianghaolu — APPROVED** [review]

**Pull request merged**